src/main/java/Bellroy/Ui.java

```diff
@@ -0,0 +1,112 @@
+package Bellroy;
+
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * The Ui class handles reading inputs from the user and printing the output to the console
+ */
+public class Ui {
+    private Scanner scanner = new Scanner(System.in);
+
+    /**
+     * prints the initial message to the user.
+     */
+    public void printWelcomeMessage() {
+        System.out.println("____________________________________________________________\n" +
+                " Hello! I'm Bellroy\n" +
+                " What can I do for you?\n" +
+                "____________________________________________________________\n");
```

Comment:
To improve the object-oriented design, consider refactoring the code by creating a private method that handles the printing of the horizontal line and message. This method could be structured to take a message as an argument and encapsulate the functionality of printing the message flanked by horizontal lines and would enhance modularity and reduces redundancy.